js/buffer_test.ts

```diff
@@ -130,6 +134,23 @@ test(async function bufferLargeByteWrites() {
   check(buf, "");
 });
 
+test(async function bufferTooLargeByteWrites() {
+  init();
+  const tmp = new Uint8Array(72);
+  const growLen = Number.MAX_VALUE;
+  const xBytes = repeat("x", 0);
+  const buf = new Buffer(xBytes.buffer as ArrayBuffer);
+  const { nread, eof } = await buf.read(tmp);
+  try {
+    buf.grow(growLen);
+  } catch (e) {
+    assertEqual(
```

Comment:
Why don't you just assert on `e.Message` and validate that `e instanceof DenoError`.